src/nvim/ex_cmds2.c

```diff
@@ -2225,14 +2210,12 @@ void ex_compiler(exarg_T *eap)
   char_u      *buf;
   char_u      *old_cur_comp = NULL;
   char_u      *p;
-  size_t bufsize;
 
   if (*eap->arg == NUL) {
     // List all compiler scripts.
-    do_cmdline_cmd("echo globpath(&rtp, 'compiler//.vim')");
-    // ) keep the indenter happy...
+    do_cmdline_cmd("echo globpath(&rtp, 'compiler/*.vim')");  // NOLINT
```

Comment:
No need to NOLINT here as well, just writing `… compiler/""*.vim` needs much less characters and no linter silencing.